.github/workflows/install.sh

```diff
@@ -1,23 +1,15 @@
 set -ex
 
 main() {
-    curl -sSf https://build.travis-ci.org/files/rustup-init.sh | sh -s -- --default-toolchain=nightly -y
-    export PATH=$HOME/.cargo/bin:$PATH
-
-    local target=
-    if [ $TRAVIS_OS_NAME = linux ]; then
-        target=x86_64-unknown-linux-musl
-        sort=sort
-    fi
+    local target=x86_64-unknown-linux-musl
+    local sort=sort
 
     cd $HOME
     git -C binaryen pull || git clone https://github.com/WebAssembly/binaryen binaryen
     cd binaryen
     cmake .
     make wasm-opt
 
-    rustup target install $TARGET
-
     cargo install -f wasm-gc
     cargo install -f wasm-bindgen-cli
```

Comment:
I believe we may not need the -f anymore. In Rust 1.41 (which this being nightly should be 1.42) it supposedly is automatically updating on cargo install (but only if it's necessary). And since we are caching, this should reduce CI build times a lot.

Reply:
Seems like this works:

First it upgrades from .57 to .58:
![https://i.imgur.com/GvA5zsD.png](https://i.imgur.com/GvA5zsD.png)

And then it refuses to upgrade any further:
![https://i.imgur.com/bSo1v1P.png](https://i.imgur.com/bSo1v1P.png)